### PR TITLE
Upgrade drone to 0.8

### PR DESCRIPTION
**Pull request by allanrenucci**
This not only upgrade drone to 0.8 but also greatly improve Drone life-cycle management. All the docker configuration is in one file for both the server and the agent.

One can simply restart drone with the `docker-compose up` command.

This is the recommended way to install Drone. http://docs.drone.io/installation/

The main difference with the previous installation is that both the server and the agent run on the same machine. I don't see a good reason for running on two machines that justify the added complexity. I guess this is up for debate.

A temporary instance of drone0.8 is running [here](http://laraquad1.epfl.ch).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,21 +75,6 @@ $ sudo su
 # docker push lampepfl/dotty:$(date +%F)
 ```
 
-The new image should now appear in <https://hub.docker.com/r/lampepfl/dotty/tags/>. The next step is to open a PR againts <https://github.com/lampepfl/dotty> to change the tag of the docker image in `.drone.yml`.
-
-### Creating the docker image on macOS ###
```
Comment (allanrenucci) [inline README.md:96]: This is no longer true. I've just installed docker from https://docs.docker.com/docker-for-mac/install/ and was able to run the previous commands.

**Comment by OlivierBlanvillain** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,77 +1,61 @@
...
-[drone](http://readme.drone.io/0.5/install/cli/) command
+[drone](http://readme.drone.io/usage/getting-started-cli/) command.
+
+### Secrets Management ###
+Drone provides the ability to store sensitive information such as passwords.
+Secrets are loaded as environment variables. For example the secret named `sonatype_user` can be
+accessed via `"$SONATYPE_USER"`. Use lower case to name your secrets and upper case to access their
+value.
+
+You can add secrests to a build via the UI on http://dotty-ci.epfl.ch or
```
Comment (felixmulder) [inline README.md]: *secrets

**Review by felixmulder — APPROVED** [review]
LGTM

**Pull request merged**